modules/nodes-cluster-worker-latency-profiles-using.adoc

```diff
@@ -138,3 +139,80 @@ $ oc get KubeControllerManager -o yaml | grep -i workerlatency -A 5 -B 5
 
 To change the medium profile to default or change the default to medium, edit the `node.config` object and set the `spec.workerLatencyProfile` parameter to the appropriate value.
 
+[id="nodes-cluster-worker-latency-profiles-using-at-creation_{context}"]
+== Using worker latency profiles at cluster creation
```

Comment:
Second procedure in a PROCEDURE module, check with Kathryn on usage.

Reply:
@sheriff-rh you are correct - I have put each procedure in it's own module.